setup.py

```diff
@@ -22,11 +22,11 @@
     install_requires=[
         'astropy>=4.0.5,<5',
         'ctapipe~=0.12.0',
-        'ctapipe_io_magic~=0.3.0',
+        'ctapipe_io_magic',
```

Comment:
Was this an issue for you? Anyway this will be changed when we will merge into master and I will make a new version of ctapipe_io_magic (v0.4.0, which I will create from the current master branch, the one working with ctapipe v0.12)

Reply:
Yes there was an issue for some reason, anyway the version 0.3.0 is for ctapipe v0.8.0 and so I modified it. Would be great if you create v0.4.0, and then I will modify it again.

Reply:
Ok, I can do it now. I will check if I need to update a bit the README.md and then create tag and release.

Reply:
Ok, it took a bit, but now ctapipe_io_magic was tagged with version 0.4.0. Also, it can be installed with pip without cloning the repo, since now I uploaded the module in PyPI.

Reply:
Thanks! Ah so it means now one doesn't need to clone both ctapipe and ctapipe_io_magic, and instead just need to create an environment, clone only magic-cta-pipe and type `pip install .`?

Reply:
Hm, actually I succeeded in installing all the software with the following steps:
$ conda env create -n magic-cta-pipe
$ git clone https://github.com/cta-observatory/magic-cta-pipe.git 
(after changing the setpy.py about the ctapipe_io_magic dependency)
$ pip install ./magic-cta-pipe/
But probably it would be better to prepare a environment.yml file to create an environment as it is done by ctapipe or lstchain?

Reply:
To have everything working as you say, indeed we would need to create an environment.yml file, which will install ctapipe 0.12 (and dependencies), ctapipe_io_magic 0.4.0 (and dependencies) and finally magic-cta-pipe

Reply:
I see, is it possible for you to create an environment.yml file? I think it would be better and easy way for the students to install the pipeline if they need to do it by themself (in the local machine, another computer, etc)

Reply:
Sure, I can do it. I will do a commit in the update-ctapipe-v0.12 branch.

Reply:
Done!